packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -68,7 +68,8 @@ const getTabStyles = stylesFactory((theme: GrafanaTheme) => {
       color: ${colors.text};
       cursor: pointer;
 
-      svg {
+      svg,
+      i.fa {
```

Comment:
Not sure if this won't break any legacy components we have, since the icons aren't used only in navigation tabs, could be smth like action icon for ex.

Reply:
This is not a global rule, only applies to tab item

Reply:
Ah, I see, my bad, didn't notice the scope.